test/Prompt/ValuePrinter/Collections.C

```diff
@@ -0,0 +1,38 @@
+//------------------------------------------------------------------------------
+// CLING - the C++ LLVM-based InterpreterG :)
+//
+// This file is dual-licensed: you can choose to license it under the University
+// of Illinois Open Source License or the GNU Lesser General Public License. See
+// LICENSE.TXT for details.
+
+//------------------------------------------------------------------------------
+
+// RUN: cat %s | %cling -Xclang -verify 2>&1 | FileCheck %s
+
+#include <string>
+#include <tuple>
+#include <vector>
+
+std::vector<bool> Bv(5,5)
+// FIXME: Printing std::vector<bool> is still broken.
+// But the line above at least tests cling doesn't crash because of it.
+// BROKENCHECK: (std::vector<bool> &) { true, true, true, true, true }
```

Comment:
No std::vector\<bool\> does not print.
The PR added that ability.
That was overruled and asked to be taken out.